profile/src/main/java/rdx/works/profile/data/model/Profile.kt

```diff
@@ -130,10 +130,17 @@ data class Profile(
 val Profile.currentGateway: Radix.Gateway
     get() = appPreferences.gateways.current()
 
-val Profile.currentNetwork: Network?
+val Profile.currentNetwork: Network
     get() {
         val currentGateway = currentGateway
-        return networks.find { it.networkID == currentGateway.network.id }
+        return networks.find {
+            it.networkID == currentGateway.network.id
+        } ?: Network(
+            networkID = Radix.Gateway.default.network.id,
+            accounts = identifiedArrayListOf(),
+            personas = identifiedArrayListOf(),
+            authorizedDapps = emptyList()
+        )
```

Comment:
here is the "trick" I do for the current network. I do not like it very much but we might think something totally different.

Reply:
the point is instead of adding the `profile.currentNetwork?.knownNetworkId ?: Radix.Gateway.default.network.networkId()` on every place in the code that we need the `knownNetworkId` we add the "?:" inside the extension

Reply:
What if instead of that (which is a bit dangerous), we added something like:

```
val Profile.currentNetworkId: NetworkId
     get() = currentNetwork?.networkId ?: Radix.Gateway.default.network.networkId()
```